AmplifyCombineSupport/AmplifyCombineSupportTests/GeneralTests.swift

```diff
@@ -0,0 +1,260 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import XCTest
+
+import AmplifyCombineSupport
+
+@testable import Amplify
+@testable import AmplifyTestCommon
+
+class StorageGeneralTests: XCTestCase {
```

Comment:
file name GeneralTest.swift -> StorageGeneralTests.swift, and it is missing from AmplifyCombineSupportTests/Storage. opening the folder in finder shows the file though

Reply:
Grr. Had some problems with Xcode losing file references when renaming. Will fix.